### DISCUSSION TITLE
Jetpack Compose support wrapper for AndroidFragmentApplication subclass

### DISCUSSION THREAD

**Discussion by kyzmitch** [libktx/ktx#478] [Ideas]
I started using original libGDX and found that it only provides view and fragment and nowadays for the modern Android app it is good to use modern Jetpack Compose framework and I created Composable which wraps the libGDX functionality and fragment using `AndroidViewBinding` and inflate functionality. 

I think it would be good to have something similar ready out of the box. The thing is that Composable should be written in Kotlin and can't be really a part of original libGDX written in Java. So that, I'm wondering if adding Jetpack Compose support to libKTX is a good idea? I haven't found similar discussions, maybe only one, but about some DSL question

**Comment by czyzby** [comment]
Sounds like a useful addition. I think it deserves a separate module such as `ktx-compose`. Would you mind submitting a PR for that? [Here's a guide](https://github.com/libktx/ktx/blob/master/.github/CONTRIBUTING.md#adding-a-new-ktx-module) on how to add a new module to the repository. Let me know if you need help. Cheers.

**Comment by charl56** [comment]
Hello, do you have any news about this module, if it has be added to libktx ?
